assets/sass/_buttons.scss

```diff
@@ -83,3 +83,8 @@
         @apply border-gray-700 text-gray-700 bg-transparent;
     }
 }
+
+.btn-scroll-top {
+    @apply text-white text-center cursor-pointer opacity-50 rounded-sm mt-8 w-8 h-4;
+    @include bg-gradient(#90a4ae,#78909c, 30em, 0, 0);
```

Comment:
Any reason not to use one of the color-palette colors here? We really should, wherever possible. 😉 🙏

Reply:
Will do! Thanks.